deploy/requirements-deploy.txt

```
@@ -0,0 +1,4 @@
+jupyterlab>=4,<5
+jupyterlite-core>=0.4,<0.5
+jupyterlite-pyodide-kernel
+.
```

Looks like we need to update to get the extension from the parent directory now:

```suggestion
..
```